### PR TITLE
Publish unshipped APIs

### PR DESCRIPTION
**Pull request by dreddy-work**
Publish pending APIs.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/5769)

### CODE REVIEW

**Comment by dreddy-work** [comment]
@RussKie, What do you do with AnalyzerReleases.Shipped and *UnShipped? It seems this is the first release we introduced them.

**Review by Tanya-Solyanik — APPROVED** [review]

**Comment by RussKie** [comment]
> What do you do with AnalyzerReleases.Shipped and *UnShipped? It seems this is the first release we introduced them.

We need to "publish" those as well.

File: src/System.Windows.Forms.Analyzers/src/System.Windows.Forms.Analyzers.csproj [hunk 1]
```diff
@@ -4,6 +4,7 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <LangVersion>Preview</LangVersion>
     <Nullable>enable</Nullable>
+    <NoWarn>$(NoWarn);RS2007</NoWarn>
```
Comment (dreddy-work) [inline src/System.Windows.Forms.Analyzers/src/System.Windows.Forms.Analyzers.csproj:7]: what is this for? why are we getting this warning?

Comment (RussKie) [reply src/System.Windows.Forms.Analyzers/src/System.Windows.Forms.Analyzers.csproj:7]:
```
C:\Repos\winforms\src\System.Windows.Forms.Analyzers\src\AnalyzerReleases.Shipped.md(1,1): error RS2007: Analyzer release file 'AnalyzerReleases.Shipped.md' has a missing or invalid release header '## Release 6.0 RC2' [C:\Repos\winforms\src\System.Windows.Forms.Analyzers\src\System.Windows.Forms.Analyzers.csproj]
```

Comment (dreddy-work) [reply src/System.Windows.Forms.Analyzers/src/System.Windows.Forms.Analyzers.csproj:7]: thanks for raising issue.

**Pull request merged**